### DISCUSSION TITLE
ReplCon

### DISCUSSION THREAD

**Discussion by Sonichigo** [Alphasians/Support#66] [Announcements]
ReplCon is being organized by Repl.it is year on following date:-
**April 2, 2022**
**10:00AM - 3:00PM PST**

To enter the ticket giveaway tell us:- 
1. Name & Email
2. Why you want to attend ReplCon 

4 Winners will be announced on 19th March.

**Comment by king-rohit777** [comment]
Rohit Jaiswal
jrohit12345400@gmail.com

My reason to attend ReplCon is to gain knowledge and upskilling. I am hoping to learn from leading industry experts and change our way of perspective towards things. I think attending ReplCon can benefit me in various ways.
